src/scicookie/hooks/post_gen_project.py

```diff
@@ -241,12 +252,24 @@ def prepare_git():
     print("=" * 80)
 
 
+def add_binding_source_files():
+    if BUILD_SYSTEM == "maturin":
+        build_system_dir = PROJECT_DIRECTORY / "build-system"
+        src_system_dir = PROJECT_DIRECTORY/ "src"
+        if USE_SRC_LAYOUT :
+            shutil.move(build_system_dir/"lib.rs", "src")
+        else:
+            os.makedir(src_system_dir)
+            shutil.move(build_system_dir / "lib.rs", src_system_dir)
+    else :
```

Comment:
```suggestion
    else:
```

actually, we don't need this `else` .. but let's keep it ... and merge this PR asap :)